ern-container-gen/src/copyRnConfigAssets.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  handleCopyDirective,
+  NativePlatform,
+  findDirectoriesHavingRnConfig,
+  shell,
+} from 'ern-core'
+import { getAssetsPath } from './getAssetsPath'
+import readDir from 'fs-readdir-recursive'
+import path from 'path'
+
+export const supportedAssetsExts = ['.ttf', '.otf']
```

Comment:
what is the context behing supporting 2 font extensions?

Reply:
As far as I know, they are the only two font formats supported by iOS/Android

Reply:
There is no clear documentation though that's the closest I found 
Android : https://developer.android.com/guide/topics/resources/font-resource?authuser=1 
iOS : https://developer.apple.com/documentation/uikit/text_display_and_fonts/adding_a_custom_font_to_your_app

Reply:
Yes, so for iOS `ttf` and `otf`. For Android doc is referring to resources kept in the `res` directory as resources, not the `assets` directory that is the target directory used by React Native. Googling around I can see that for `ttc` support in assets you need some extra code. Should be good for `ttf` and `otf`, these are the most common font types anyway